<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModulatorCell</class>
 <widget class="QWidget" name="ModulatorCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="2">
    <widget class="ModulatorComboSrc" name="comboSource1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="9">
    <widget class="QLabel" name="labelFinalRange">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="ModulatorComboCurve" name="widgetShape2" native="true">
     <property name="maximumSize">
      <size>
       <width>53</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ModulatorComboSrc" name="comboSource2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6">
    <widget class="QSpinBox" name="spinAmount">
     <property name="prefix">
      <string>Ã— </string>
     </property>
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="ModulatorComboCurve" name="widgetShape1" native="true">
     <property name="maximumSize">
      <size>
       <width>53</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="ModulatorComboDest" name="comboDestination">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QComboBox" name="comboTransform">
     <item>
      <property name="text">
       <string>no operation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>absolute value</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelModNumber">
       <property name="text">
        <string>Modulator
#1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDetails">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModulatorComboCurve</class>
   <extends>QWidget</extends>
   <header>modulatorcombocurve.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModulatorComboSrc</class>
   <extends>QComboBox</extends>
   <header>modulatorcombosrc.h</header>
  </customwidget>
  <customwidget>
   <class>ModulatorComboDest</class>
   <extends>QComboBox</extends>
   <header>modulatorcombodest.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
